modules/consignment-api/ecs.tf

```diff
@@ -1,6 +1,9 @@
 locals {
   app_port = 8080
 }
+
+data "aws_caller_identity" "current" {}
```

Comment:
Is it worth putting this in a `data.tf`, to keep things consistent?

Reply:
Yeah, should do, I was being lazy.